nfd-worker-job.yaml.template

```diff
@@ -14,6 +14,16 @@ spec:
     spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - topologyKey: kubernetes.io/hostname
+              labelSelector:
+              matchExpressions:
```

Comment:
You need to add one more indent on the `matchExpressions` field

Reply:
@marquiz I don't know if you missed this one

Reply:
> @marquiz I don't know if you missed this one

Yup, seems I did. Thanks for spotting this! Fixed, now